src/main/java/lesson17/OfficeDAOImpl.java

```diff
@@ -38,4 +39,52 @@
         OJDBCUtils.closeAllCloseble(conn, stmt, rs);
         return offices;
     }
+
+    @Override
+    public boolean insertOffice(Offices office) throws SQLException {
+        Connection conn = OJDBCUtils.getConnection();
+        PreparedStatement stmt = conn.prepareStatement("insert into offices(OFFICE, CITY, REGION, MGR, TARGET, SALES)" +
+                " values (?, ?, ?, ?, ?, ?)");
+
+        setOfficeValuesToStatement(office, stmt);
+
+        boolean isOk = stmt.executeUpdate() > 0;
+        OJDBCUtils.closeAllCloseble(stmt, conn);
```

Comment:
Cool...

Reply:
It isn't help you if we have exception. try{}finally{}